core.py

```diff
@@ -0,0 +1,38 @@
+from collections import defaultdict
+from listener import Listener
+from pos import Pos
+from button import Button
+from listlayout import ListLayout
+from text import Text
+
+import pygame
+import events
+import tooltip
+import colour
+
+
+class Core(object):
+    def __init__(self):
+        self.all_ui = pygame.sprite.LayeredDirty()
+        Button.groups = self.all_ui
```

Comment:
This paradigm is interesting and maybe dangerous, every core instance will reset the element's `groups` class member. It is kind of like making a global change when u instantiate something locally. This behaviour is very odd and can perhaps lead to strange bugs down the line.